# Copyright (c) 2014 David Black. Licensed as GPLv3.
# See https://gitorious.org/asdgasd/my-fdroiddata
Use consistent appname which is used on website: http://code.google.com/p/earth-live-wallpaper/

diff --git a/res/values-es/strings.xml b/res/values-es/strings.xml
index a3f9b91..5c89079 100644
--- a/res/values-es/strings.xml
+++ b/res/values-es/strings.xml
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 	<string name="hello">Hello World, SLWP!</string>
-	<string name="app_name">EarthRot</string>
 	<string name="description">rotación Globe (OpenGL VBO)</string>
 	<string name="p_speed">velocidad</string>
 	<string name="p_speedsum">Elija la velocidad de rotación de la Tierra
diff --git a/res/values-fr/strings.xml b/res/values-fr/strings.xml
index 7d1e443..32d857c 100644
--- a/res/values-fr/strings.xml
+++ b/res/values-fr/strings.xml
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 	<string name="hello">Hello World, SLWP!</string>
-	<string name="app_name">EarthRot</string>
 	<string name="description">Globe terrestre rotatif (Opengl VBO)</string>
 	<string name="p_speed">Vitesse de rotation</string>
 	<string name="p_speedsum">Choisissez la vitesse de rotation du globe</string>
diff --git a/res/values/strings.xml b/res/values/strings.xml
index 501db7e..99d20aa 100644
--- a/res/values/strings.xml
+++ b/res/values/strings.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 	<string name="hello">Hello World, SLWP!</string>
-	<string name="app_name">EarthRot</string>
+	<string name="app_name">Earth Live Wallpaper</string>
 	<string name="description">Rotating earth globe (Opengl VBO)</string>
 
 
